getting-started/pom.xml

```diff
@@ -48,6 +48,8 @@
     </dependencies>
 
     <build>
+        <!-- required because we can't use a *-runner.jar in .s2i/environment's JAVA_APP_JAR -->
+        <finalName>${project.artifactId}</finalName>
```

Comment:
It creates an issue with the getting started guide that does not use this, and so it contains the path to the "long" name. Can we use the "long" name in the `environment` file?

Reply:
@cescoffier  - check my comment above a technique I followed in my builder image to compute the actual artifactName.

Not sure how much that will help in this context but just a thought

Reply:
Yes, because it requires buildah right?

Reply:
no. i just used in a script to find the final artifact name.. it can be used with any context vanilla docker, buildah, kaniko etc

Reply:
> It creates an issue with the getting started guide that does not use this

You mean we have we hard-coded somewhere else (where?) that we expect the built JAR to have a version? That won't work because we can't use a *-runner.jar in .s2i/environment's `JAVA_APP_JAR` ... then we either have to find another way (I briefly played with `S2I_SOURCE_DEPLOYMENTS_FILTER` in #90 [before squashing](https://github.com/quarkusio/quarkus-quickstarts/compare/master...vorburger:issue-1345_s2i-HISTORY-BEFORE-SQUASH), but it didn't work - perhaps that's worth another attempt...), or work on a patch to https://github.com/jboss-container-images/openjdk ... phew.

Reply:
I think I've figured out a solution for this (see 9adaa4b22bb4a1a8ca785329a2799ecc665211fc)... just need to fully re-test it e2e ...

Reply:
Resolving this conversation, as 9adaa4b indeed does the trick and works great.